tests/security/test_auth.py

```diff
@@ -0,0 +1,181 @@
+"""
+Tests for auth.py
+"""
+
+import pytest
+from connexion.exceptions import Unauthorized
+from foca.security.auth import param_pass
+from unittest.mock import MagicMock
+
+
+"""
+Test for checking authorization requirement
```

Comment:
Please add those as docstrings to the function headers as described in [PEP257](https://www.python.org/dev/peps/pep-0257/). For example:

```py
def my_func():
    """Docstring describing what my_func() does"""
    <function body>
```

Reply:
I have done this and also fixed a typo (changed `useinfo` to `userinfo`) in the latest commit

Reply:
I was about to commit the same :p

Reply:
Sorry :-S

Reply:
https://github.com/elixir-cloud-aai/foca/tree/dev/tests/config I'll correct these files soon. 😅